python/versionifier.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+from setuptools.extern.packaging.version import Version, InvalidVersion
+
+def remove_prefix(s, prefix):
```

Comment:
A reason to upgrade to 3.9 if there ever was one.

Reply:
We only have this in 2 places (3 if you include the C++ `nonstd` version...). Once its redefined in >5% of our files, then we upgrade/burn it all down.

Reply:
Joking aside, we have the suffix variant in quite a few more places. Still not a reason :)